### PR TITLE
Disabling TF32 for accuracy comparison

### PR DESCRIPTION
**Pull request by anijain2305**
Some context here - https://github.com/pytorch/torchdynamo/issues/611

Even though this is not enough, it can help reduce the number of spurious models that fail because of TF32 issues.

@jansel @Chillee @ngimel Does this make sense?

### CODE REVIEW

File: benchmarks/common.py [hunk 1]
```diff
@@ -48,6 +49,26 @@
 output_filename = None
 
 
+# We are primarily interested in tf32 datatype
+torch.backends.cuda.matmul.allow_tf32 = True
```
Comment (jansel) [inline benchmarks/common.py:53]: We might want to set this inside `main()`.  A lot of the benchmarks change global state when we import them.

File: benchmarks/common.py [hunk 2]
```diff
@@ -667,53 +688,56 @@ def run_one_model(
...
+                    if not same(correct_result, correct_rerun_result):
+                        print("INCORRECT - Variation in Eager runs itself")
+                        if not skip_accuracy_check:
+                            return sys.exit(-1)
+
+                torch.manual_seed(1337)
+                torchdynamo.reset()
+                if experiment.func is cold_start_experiment:
+                    results = []
+                    results.append(experiment(model, example_inputs, optimize_ctx))
```
Comment (jansel) [inline benchmarks/common.py:719]: Won't this disable tf32 for our performance measurements?

**Review by jansel — CHANGES_REQUESTED** [review]

File: benchmarks/common.py (hunk 2 above)
Comment (anijain2305) [reply benchmarks/common.py]: This one is specific to cold_start_experiment. The real experiment is called later on.
But, I am moving this one out of scope of with no_tf32.

File: benchmarks/common.py (hunk 1 above)
Comment (anijain2305) [reply benchmarks/common.py]: Good idea.

**Comment by anijain2305** [comment]
Closing as this does not really help.

**Pull request closed without merging**